scenarios/lb_monitoring/resources/vm.tf

```diff
@@ -0,0 +1,83 @@
+locals {
+  workspace_prefix = terraform.workspace == "default" ? "" : terraform.workspace
+}
+
+resource "opentelekomcloud_compute_keypair_v2" "kp" {
+  name       = var.key_pair.key_name
+  public_key = var.key_pair.public_key
+}
+
+data "opentelekomcloud_images_image_v2" "current_image" {
+  name        = var.ecs_image
+  most_recent = true
+}
+
+# Create security group for instances
+resource "opentelekomcloud_compute_secgroup_v2" "lb_group" {
+  description = "Allow external connections to ssh and http ports"
+  name        = "${var.scenario}_grp"
+  rule {
+    cidr        = "0.0.0.0/0"
+    from_port   = 22
+    ip_protocol = "tcp"
+    to_port     = 22
+  }
+  rule {
+    cidr        = "0.0.0.0/0"
+    from_port   = 80
+    ip_protocol = "tcp"
+    to_port     = 80
+  }
+  rule {
+    cidr        = "0.0.0.0/0"
+    from_port   = 8080
+    ip_protocol = "tcp"
+    to_port     = 8080
+  }
```

Comment:
I'm not sure, but let's add icmp rule here
```
  rule {
    cidr        = "0.0.0.0/0"
    from_port   = -1
    ip_protocol = "icmp"
    to_port     = -1
  }
```

Reply:
let's leave that without icmp rule